### ISSUE TITLE
Wall post and timestamp class names incorrect

### ISSUE THREAD

**Issue by chasefarmer2808** [weskerfoot/DeleteFB#42]
Just ran this and was getting `SELENIUM_EXCEPTIONS` when it tried to get the post and timestamp elements.  Did some debugging in Pycharm and saw the error message was `No element found for selector "_5_jv"` and `No element found for selector "timestamp"`.  Looks like the class names have changed.  I set the `post_content_sel` variable to `"_5pcb"` and the `post_timestamp_sel` variable to `"timestampContent"` and the program began to work as expected.  I got the new class names from inspecting the elements on the page.  Not sure if they are the same for everyone, but this was the case for myself and a friend's account.

**Comment by weskerfoot** [comment]
Thanks for reporting. I'll look into it.

**Comment by weskerfoot** [comment]
Yes they seem to have changed for me as well.

**Comment by weskerfoot** [comment]
```
<abbr title="28/05/2019, 23:33" data-utime="1559100825" data-shorten="1" class="_5ptz timestamp livetimestamp">
<span class="timestampContent" id="js_wy">1 min</span>
</abbr>
```

This is what I'm seeing for the timestamp element. Can you check if you have something similar?

You can check (in Firefox) by right clicking on the timestamp, inspect element, then right click on the HTML in the inspect and "Copy" -> "Copy outer HTML".

**Comment by weskerfoot** [comment]
You may be getting a different version of the site, which is an interesting problem to try and solve.

**Comment by weskerfoot** [comment]
I think `timestampContent` should work for both versions, and potentially `userContent` for the post content.

**Comment by AstraQuinn** [comment]
@weskerfoot this resolved it for me

**Comment by weskerfoot** [comment]
Should be fixed in https://github.com/weskerfoot/DeleteFB/commit/1241d6e7e89863d95f037c350722dd52a6aa44f8

Will do a new release. Thanks for reporting!

Can you let me know if the above commit fixes the issues for you?

**Comment by weskerfoot** [comment]
I've released version `1.1.1` to PyPI which should hopefully fix these issues.
